albatross_parameters.m

```diff
@@ -1,17 +1,24 @@
 %% Call all Albatross Parameters
 
-%Wing + Aero Parameters
+%Wing Geometry
 
 S   = 68;        %m2
 AR  = 18;            
 b   = sqrt(S*AR); %m
 c   = S/b;
 
-
+% Aerodynamic Parameters
 e   = 0.85;
-cdo = 0.03;

```

Comment:
Ben's code needs the cdo variable. We'll update that file as well when this change is made.

Reply:
oh ok sorry didnt realise. Whats the difference between cd0 and cdo?

Reply:
cdo decreases when payload dropped, but since its iterating in Bens code, in the next loop, it needs to start with cd0